### DISCUSSION TITLE
blog wrapping is much simpler to use than github pages and we don't want to leave the github structure

### DISCUSSION THREAD

**Discussion by guiseek** [community/community#2920] [Discussions]
many people use markdown to write their articles for their blogs (or not), and end up being published here, however, as markdown. Despite rendering the markdown, we miss a structure similar to this new one for discussions like wrapping, allowing a good navigation, search and even a friendly url to be found in the search engines.

It would also be interesting to be able to link code examples like gists and even create a mini repository as an example to demonstrate what was explained in the article.

What do you think of the idea? count on me for more insights or help with code development, ok?

**Comment by AgrMayank** [comment]
The second point could be really helpful when discussing errors and giving code examples to the users.
